week-1/K-functions-parameters/exercise3.js

```diff
@@ -1,5 +1,8 @@
 // Write your function here
+function createGreeting(){
+return "Hello! my name is"
 
-var greeting = createGreeting("Daniel");
+   
+var greeting = createGreeting + "Daniel";}
```

Comment:
Having the `}` here at the end means that the `greeting` variable belongs to the function - and it shouldn't.  In fact, are you sure your code runs here? Visual Studio should underline your code in red if it thinks there are errors.

Rethink what's happening here :) Remember that `createGreeting` is a function, not a variable and that it should take a name (a string) as parameter and return a greeting